src/hooks/useDropdownAutoHeight.ts

```diff
@@ -2,22 +2,21 @@
 
 import { MutableRefObject, useLayoutEffect, useState } from 'react';
 
```

Comment:
```suggestion
type MaxHeightObject = {
   toTop?: number;
   toBottom: number | 'auto';
}
```

Reply:
This type i used in two places (potentially more in the future) so would be nice to make a `type` of it.